### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'target_tensor'

### ISSUE THREAD

**Issue by DogeWatch** [pytorch/pytorch#8275]
here is my code:
`import torch
import torch.utils.data as Data
x = torch.linspace(1,10,10)
 y = torch.linspae(10,1,10)
dataset = Data.TensorDataset(data_tensor=x,target_tensor=y)
`
and it give my a TypeError: __init__() got an unexpected keyword argument 'target_tensor'
my python version is 2.7.15 and torch version is 0.4.0

**Comment by vishwakftw** [comment]
From the [docs](https://pytorch.org/docs/stable/_modules/torch/utils/data/dataset.html#TensorDataset), there is no keyword called `target_tensor` or `data_tensor`. You can pass a variable number of arguments to the constructor
```python
x = torch.linspace(1, 10, 10)
y = torch.linspace(10, 1, 10)
dataset = Data.TensorDataset(x, y)
```

**Comment by fmassa** [comment]
For information, this was changed in 0.4. We used to have two arguments, but now we have the `*args`.

**Comment by fjz15056311771** [comment]
i dont think this change is friendly for user! this small question make me confuse for a long time

**Comment by mundanePeo** [comment]
this change cannot solve the problem.
